### PR TITLE
csv generation fixes

### PR DESCRIPTION
**Pull request by yunusdawji**
this generates csv file and very basic load data function

### CODE REVIEW

File: process.py [hunk 1]
```diff
@@ -74,24 +82,75 @@ def process_all():
     """
 
     images = []
-    image_numpys = []
-    IMAGE_PATH = 'C:\\Samples\\ImageProcessor\\Images'
+    image_numpys = np.empty([10000 , 1])
```
Comment (magdapoppins) [inline process.py:85]: Why 10 000, 1?

File: process.py [hunk 2]
```diff
@@ -1,7 +1,14 @@
+import matplotlib
+matplotlib.use('TkAgg')
```
Comment (magdapoppins) [inline process.py:2]: We really need this? (TkAgg?)

**Review by magdapoppins — COMMENTED** [review]
Just a few questions. Nice job with the load_data()!

File: process.py (hunk 1 above)
Comment (yunusdawji) [reply process.py:85]: When u store a 100*100 img in one row it becomes 10000*1. This makes storing the imgs easier

File: process.py (hunk 2 above)
Comment (yunusdawji) [reply process.py:2]: There was some issue with mac and virtenv, and this was a workaround.

Comment (magdapoppins) [reply process.py:2]: Strange. Issue in the image rendering..?

Comment (yunusdawji) [reply process.py:2]: Yes. Its an issue with image rendering. This might not be needed for windows

Comment (magdapoppins) [reply process.py:2]: Ok.

**Pull request merged**